File: android/app/src/main/java/com/stopbet/app/BlockActivity.java
package com.stopbet.app;

import android.app.Activity;
import android.os.Bundle;
import android.view.KeyEvent;
import android.widget.TextView;

public class BlockActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        TextView tv = new TextView(this);
        tv.setText("🚫 ACESSO BLOQUEADO\n\nVolte em 12 horas.");
        tv.setTextSize(22);
        tv.setPadding(40, 200, 40, 40);

        setContentView(tv);
    }

    // ⛔ Bloqueia botão voltar
    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        if (keyCode == KeyEvent.KEYCODE_BACK) {
            return true;
        }
        return super.onKeyDown(keyCode, event);
    }
}

File: android/app/src/main/java/com/stopbet/app/BlockActivity.kt
package com.stopbet.app

import android.app.Activity
import android.os.Bundle
import android.view.KeyEvent
import android.view.WindowManager

class BlockActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        window.addFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN or
            WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON or
            WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL
        )
    }

    override fun onBackPressed() {
        // botão voltar bloqueado
    }

    override fun onKeyDown(keyCode: Int, event: KeyEvent?): Boolean {
        return when (keyCode) {
            KeyEvent.KEYCODE_HOME,
            KeyEvent.KEYCODE_APP_SWITCH -> true
            else -> super.onKeyDown(keyCode, event)
        }
    }
}
